device/device.go

```diff
@@ -178,8 +178,31 @@ func (d *Device) IsConnected() bool {
 }
 
 // AddInterface adds an interface to the device
```

Comment:
While you're at this, you might want to extend this documentation detailing this should be called with an interface loaded with `ParseInterface`, errors, etc...